src/helpers/config.ts

```diff
@@ -390,6 +400,16 @@ export const allCoinMetas: Record<string, CoinMetadata> = {
     Icon: 'polygon',
     value: 'tpolygon',
   },
+  tpolygonToken: {
+    Title: 'TPOLYGON TOKEN',
+    Description: 'Testnet Polygon Token',
+    Icon: 'polygon',
+    value: 'tpolygonToken',
+    ApiKeyProvider: 'polygonscan.com',
+    minGasLimit: '400,000',
+    defaultGasLimit: '1,000,000',
+    defaultGasLimitNum: 1000000,
```

Comment:
For polygon keep these values same as eth

Reply:
Also we need to have a min value of 30 gwei for polygon, can you please make that change

Reply:
addressed